Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml

```diff
@@ -74,6 +74,25 @@
   - type: Appearance
   - type: WiresVisuals
 
+- type: entity
+  parent: VendingMachine
+  id: VendingMachineCondiments
+  name: Condiment Station
+  description: Slather these thick gooey substances on your food for a full flavor effect.
+  components:
+  - type: Sprite
+    sprite: Structures/dispensers.rsi
+    drawdepth: SmallObjects
+    state: condiment
+    netsync: false
+  - type: Advertise
+    pack: AmmoVendAds
```

Comment:
Please give this its own advertisements, as of right now you'll have a condiment machine advertising about ammo. You can put the adverts into their own FTL file.

Reply:
Yep, that was a placeholder I forgot to replace. My bad.